.clang-tidy

```diff
@@ -1,5 +1,5 @@
 ---
-Checks:          '-clang-diagnostic*,-clang-analyzer*,modernize-use-nullptr'
```

Comment:
Why do we remove `modernize-use-nullptr`?!

Reply:
a mistake I'd say. I'll restore it (or you can do it) directly in master once I'm done with checks on another PR.